packages/components/src/components/domainproperties/issues/actions.ts

```diff
@@ -6,12 +6,13 @@ import { Principal } from '../../..';
 
 import { IssuesListDefModel } from './models';
```

Comment:
I noticed this while perusing related files but the `IssuesListDefModel`'s return type for `getOptions()` is incorrect. It doesn't return an `Immutable.Record` but rather an immutable (by `immer`) subset object of a `IssuesListDefModel`. May want to define another interface using `Exclude` or `Pick`.

Reply:
This was addressed in a different, recently merged PR: https://github.com/LabKey/labkey-ui-components/pull/252